README.md

```diff
@@ -3,6 +3,10 @@ Node Airbrake
 
 [![Circle CI](https://circleci.com/gh/airbrake/node-airbrake.svg?style=shield)](https://circleci.com/gh/airbrake/node-airbrake)
 [![npm version](https://badge.fury.io/js/airbrake.svg)](https://badge.fury.io/js/airbrake)
+[![Documentation Status](http://inch-ci.org/github/airbrake/node-airbrake.svg?branch=master)](http://inch-ci.org/github/airbrake/node-airbrake)
```

Comment:
This badge appears blank, is that expected?

![](https://api.monosnap.com/rpc/file/download?id=kFFNkMbkF5I0FUK6SoQUOOWlSzBF41)